header.yaml

```diff
@@ -0,0 +1,34 @@
+default-headers: 
+  - name: avax
+    header: |
+      // Copyright (C) 2019-2023, Ava Labs, Inc. All rights reserved.
```

Comment:
is 2023 intended here ?

Reply:
yes
all avax files headers are dated with 2023 in cortina 15